models.js

```diff
@@ -1,17 +1,20 @@
-const database = psql => require('./db/connection.js').query(psql)
+const database = (psql, ...replacements) => require('./db/connection.js').query(psql, ...replacements)
 
 exports.selectTopics = () => database(`SELECT * FROM topics;`)
     .then(({ rows: topics }) => topics)
 
 exports.selectArticles = () => database(`
-    SELECT articles.author, articles.article_id, articles.title, articles.topic, articles.created_at, articles.votes, 
-    CAST (COUNT(comment_id) AS INT) AS comment_count
-    FROM articles
-    LEFT JOIN comments
-    ON articles.article_id = comments.article_id
-    GROUP BY articles.article_id
-    ORDER BY articles.created_at DESC
-    ;`)
-    .then(({ rows: articles }) => {
-        return articles
-    })
\ No newline at end of file
+SELECT articles.author, articles.article_id, articles.title, articles.topic, articles.created_at, articles.votes, 
+CAST (COUNT(comment_id) AS INT) AS comment_count
+FROM articles
+LEFT JOIN comments
+ON articles.article_id = comments.article_id
+GROUP BY articles.article_id
+ORDER BY articles.created_at DESC;`)
+    .then(({ rows: articles }) => articles)
+
+exports.selectArticlesById = (articleId) => database(`
```

Comment:
We're fetching a single article, so this function name (which is plural) is misleading, a more accurate name would be `selectArticleById`